### PR TITLE
feat: Add new catalog cards to My List, Accounts and HomePage

### PR DESCRIPTION
**Pull request by juanmahidalgo**
This PR includes:

### Trendings
image
### Newest
image

## My Lists

image

## Accounts page

image

Also: 
- When an item is already in the state and it's requested again, the success action will merge the results (to preserve any extra data that is missing in the second request). This happens when the catalog is fetched and then an item is fetched using the Item API .

### CODE REVIEW

**Review by flobarreto — APPROVED** [review]

File: webapp/src/modules/favorites/sagas.ts [hunk 1]
```diff
@@ -161,19 +163,20 @@ export function* favoritesSaga(getIdentity: () => AuthIdentity | undefined) {
...
 
       if (results.length > 0) {
-        const result: { data: Item[] } = yield call(
-          [itemAPI, 'get'],
-          options.filters
-        )
+        const result: { data: Item[] } = yield call([catalogAPI, 'get'], {
+          first: results.length,
+          ids
+        })
```
Comment (kevinszuchet) [inline webapp/src/modules/favorites/sagas.ts:179]: Why not use the `options.filters` property that already has what's needed?

File: webapp/src/modules/favorites/sagas.ts [hunk 2]
```diff
@@ -43,18 +43,20 @@ import {
...
     retryDelay: retryParams.delay,
     identity: getIdentity
-  })
-
+  }
   const favoritesAPI = new FavoritesAPI(MARKETPLACE_FAVORITES_SERVER_URL, {
     retries: retryParams.attempts,
     retryDelay: retryParams.delay,
     identity: getIdentity
   })
```
[the remark this answers is not in the document]
Comment (juanmahidalgo) [reply webapp/src/modules/favorites/sagas.ts]: ops, I intented to use it there as well, good catch!

File: webapp/src/modules/favorites/sagas.ts (hunk 1 above)
Comment (juanmahidalgo) [reply webapp/src/modules/favorites/sagas.ts]: sure, done!

File: webapp/src/modules/item/reducer.spec.ts [hunk 3]
```diff
@@ -207,19 +230,42 @@ describe.each([
...
     loading: loadingReducer([], requestAction)
   }
 
-  it('should return a state with the the loaded items with the fetched item and the loading state cleared', () => {
-    expect(itemReducer(initialState, successAction)).toEqual({
-      ...INITIAL_STATE,
-      loading: [],
-      data: { ...initialState.data, [item.id]: item }
+  describe('and the fetched item is not in the state', () => {
+    it('should return a state with the the loaded items with the fetched item and the loading state cleared', () => {
```
Comment (LautaroPetaccio) [inline webapp/src/modules/item/reducer.spec.ts:242]: What do you think about this change? Is it easier to read?
```suggestion
    it('should return a state with the loaded items, the fetched item and the loading state cleared', () => {
```

File: webapp/src/modules/item/reducer.spec.ts [hunk 4]
```diff
@@ -207,19 +230,42 @@ describe.each([
...
+      minPrice: '1234'
+    }
+    beforeEach(() => {
+      initialState = {
+        ...INITIAL_STATE,
+        data: { anotherId: anotherItem, [item.id]: item },
+        loading: loadingReducer([], requestAction)
+      }
+      successAction = fetchItemSuccess({ ...item, ...newItemData })
+    })
```
Comment (LautaroPetaccio) [inline webapp/src/modules/item/reducer.spec.ts:262]: Would you mind defining the newItemData inside of the before each?
```suggestion
    let newItemData: Item
    beforeEach(() => {
      newItemData = {
        minPrice: '1234'
      } as Item
      initialState = {
        ...INITIAL_STATE,
        data: { anotherId: anotherItem, [item.id]: item },
        loading: loadingReducer([], requestAction)
      }
      successAction = fetchItemSuccess({ ...item, ...newItemData })
    })
```

File: webapp/src/modules/item/reducer.spec.ts [hunk 5]
```diff
@@ -207,19 +230,42 @@ describe.each([
...
+    it('should return a state with the old items merged with the new items with the fetched item and the loading state cleared', () => {
```
Comment (LautaroPetaccio) [inline webapp/src/modules/item/reducer.spec.ts:263]: What do you think about this change? Is it easier to read?
```suggestion
    it('should return a state containing the old items merged with the new fetched item and the loading state cleared', () => {
```

**Review by LautaroPetaccio — COMMENTED** [review]
Looks great! I've just left some comments about some tests to review.

File: webapp/src/modules/item/reducer.spec.ts (hunk 4 above)
Comment (juanmahidalgo) [reply webapp/src/modules/item/reducer.spec.ts]: sure, done!

[the remark this answers is not in the document]
Comment (juanmahidalgo) [reply webapp/src/modules/item/reducer.spec.ts]: done!

**Review by LautaroPetaccio — APPROVED** [review]
Ship it 🛥️

**Review by LautaroPetaccio — APPROVED** [review]
Ship it 🛥️

**Review by LautaroPetaccio — APPROVED** [review]
Ship it 🛥️

**Review by LautaroPetaccio — APPROVED** [review]
Ship it 🛥️

**Pull request merged**